docs/wizard.md

```diff
@@ -0,0 +1,102 @@
+### The wizard, or start command
+
+Once running the command you should first see one of the prompts below:
+
+- if you don't have an active `project`.
```

Comment:
The first thing that is being checked is whether you are logged in or not. If not then there is the ```sign-up, login``` screen first.